SPECS-EXTENDED/rust-coreutils/rust-coreutils.spec

```diff
@@ -0,0 +1,151 @@
+Summary:        Rust reimplementation of GNU core utilities
```

Comment:
Before we get into the review and clean-up, could you provide more details about this change? Why is it needed? What's the scenario we're trying to support?